src/Redux/api.tsx

```diff
@@ -428,6 +444,9 @@ const routes = {
   },
   getDailyReports: {
     path: "/api/v1/consultation/{consultationId}/daily_rounds/",
+    method: "GET",
+    TBody: Type<GetDailyReportsBody>(),
+    TRes: Type<PaginatedResponse<DailyRoundsModel[]>>(),
```

Comment:
Are you sure that this is the correct type?
```suggestion
    TRes: Type<PaginatedResponse<DailyRoundsModel>>(),
```